### PR TITLE
Cleanup for FullscreenImageActivity

### PR DESCRIPTION
**Pull request by rokasjasonas**

### CODE REVIEW

**Comment by mjurkus** [comment]
Care to convert to Kotlin?

**Comment by rokasjasonas** [comment]
https://media.giphy.com/media/3o7abnZIGzKF8OjnEI/giphy.gif

File: app/src/main/java/lt/vilnius/tvarkau/FullscreenImageActivity.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import android.os.Bundle
+import android.view.MenuItem
+import android.view.View
+import kotlinx.android.synthetic.main.activity_fullscreen.*
+import lt.vilnius.tvarkau.views.adapters.FullscreenImagesPagerAdapter
+
+class FullscreenImageActivity : BaseActivity() {
+
+    private var initialImagePosition: Int = 0
```
Comment (mjurkus) [inline app/src/main/java/lt/vilnius/tvarkau/FullscreenImageActivity.kt:11]: This approach works better - you can see where value is retrieved from:

```
private val initialImagePosition: Int
  get() = intent.getIntExtra(....)
```

Same with `photos`

**Review by mjurkus — CHANGES_REQUESTED** [review]

**Comment by rokasjasonas** [comment]
Requested changes done!

**Comment by mjurkus** [comment]
Hmm. Something went very wrong here... Unnecessary commits were added to your PR. Please, rebase.

**Comment by ViliusKraujutis** [comment]
@rokasjasonas ping

**Comment by vycius** [comment]
Closing PR due to inactivity. @rokasjasonas Feel free to reopen it if you continue working on this.

**Pull request closed without merging**